tests/utility/hash_test.cu

```diff
@@ -195,4 +195,95 @@ TEMPLATE_TEST_CASE_SIG("Static vs. dynamic key hash test",
     CHECK(hash(key) ==
           hash.compute_hash(reinterpret_cast<std::byte const*>(&key), sizeof(key_type)));
   }
+}
+
+template <typename Hash>
+__host__ __device__ bool check_murmurhash3_128_result(typename Hash::argument_type const& key,
+                                                      std::uint64_t seed,
+                                                      typename Hash::result_type expected) noexcept
+{
+  Hash h(seed);
+  return (h(key) == expected);
+}
```

Comment:
This probably can be unified with `check_hash_result`
```suggestion
```

Reply:
`check_hash_result` uses `seed` as `Hash::argument_type`, which would not work for 128bit hash, as it needs seed as `std::uint64_t`.